src/main/java/org/p2p/solanaj/rpc/types/config/TransactionConfig.java

```diff
@@ -0,0 +1,13 @@
+package org.p2p.solanaj.rpc.types.config;
+
+import lombok.Setter;
+
+@Setter
+public class TransactionConfig {
+
+    private String encoding = "json";
+
+    private int maxSupportedTransactionVersion = 0;
+
+    private String commitment;
```

Comment:
question: `commitment` is type `String` here, shouldn't it be limited to `Commitment` enum values only?

Reply:
👍 I think moshi json library work with enums out of the box when params are serialized.

Reply:
I also wanted to stay consistent with the rest of the codebase, see https://github.com/blockchain/solanaj/blob/1d7dd3d6c86e7ced2291e3e4576a4a8432c144b7/src/main/java/org/p2p/solanaj/rpc/types/config/RpcEpochConfig.java#L10 for example